Client/Extensions/EnvironmentValues+Extension.swift

```diff
@@ -0,0 +1,21 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import SwiftUI
+import Shared
+import Common
+
+extension EnvironmentValues {
+    public var themeType: SwiftUITheme {
+        let themeManager: ThemeManager = AppContainer.shared.resolve()
+
+        guard themeManager.currentTheme.type != .dark else {
```

Comment:
Why do we need the guard here for `.dark`? We can't just return `SwiftUITheme(theme: themeManager.currentTheme)` ?

Reply:
I added it because for some reason resolve was crashing but it isn't anymore so good to get rid of it as well 🤷🏼‍♂️